src/index.ts

```diff
@@ -1,17 +1,59 @@
 import { Command } from 'commander';
 import { description, version } from '../package.json';
-import commands from './commands';
+
+function isObject(value: unknown): value is object {
+  return typeof value === 'object';
+}
+
+function isFunction(value: unknown): value is () => unknown {
+  return typeof value === 'function';
+}
+
+async function getFunctionByName(functionName: string): Promise<() => unknown> {
+  const { faker } = await import('@faker-js/faker/locale/en');
+  for (const module of Object.values(faker)) {
+    if (!isObject(module)) {
+      continue;
+    }
+
+    for (const entry of Object.values(module)) {
+      if (!isFunction(entry)) {
+        continue;
+      }
+
+      // we need to use "includes" since the function names are "bound functionName"
+      if (!entry.name.includes(functionName)) {
+        continue;
+      }
```

Comment:
You could rewrite this to:

````ts
const entry = module[functionName];

if (isFunction(entry)) {
  return entry;
}
````

Reply:
While this works for now, I had planed to make the argument case insensitive in the future. So you could also write `npx faker firstname`, which is incorrect as the function name would be `firstName` (with an uppercase "N").

Reply:
How about a `(lowerModuleName, lowerFunctionName) -> import/fn` lookup table?
(Effectively, `(moduleName, functionName)` -> ``lookup[`${moduleName.toLowercase()}.${functionName.toLowercase}`]`` -> `faker.module.function`/`import fn  from @faker-js/faker/modules/module/function`)

That way you could also add aliases such as name -> person.

Reply:
I changed the way I access keys by iterating over `Object.entries`. In the future, this will allow for more preprocessing on module and function keys before comparing them to the input value. Additionally, this resolves the problem where the function name was `bound functionName`.